### PR TITLE
DEPLOY-858: List of deployed components in README.md

### PR DESCRIPTION
**Pull request by flv22**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,29 @@
 
 Check [prerequisites section](https://github.com/Alfresco/alfresco-dbp-deployment/blob/master/README-prerequisite.md) before you start.
 
+This deployment comprises:
+
+#####Core products
+- Alfresco Process Services 1.10
+- Alfresco Process Services Admin 1.10
+- Alfresco DBP Repository 6.1.1
```
Comment (cristina-sirbu) [inline README.md]: Alfresco Content Repository

File: README.md [hunk 2]
```diff
@@ -2,6 +2,29 @@
...
+- Alfresco DBP Share 6.1.1
```
Comment (cristina-sirbu) [inline README.md]: Alfresco Content Share

File: README.md [hunk 3]
```diff
@@ -2,6 +2,29 @@
...
+- Alfresco Digital Workspace 1.2.0
+- Alfresco Process Workspace 1.3.4
+- Alfresco Service Sync 3.1.2
```
Comment (cristina-sirbu) [inline README.md]: Alfresco Sync Service

File: README.md [hunk 4]
```diff
@@ -2,6 +2,29 @@
...
+- Alfresco Process Services 1.10
+- Alfresco Process Services Admin 1.10
+- Alfresco Content Repository 6.1.1
+- Alfresco Content Share 6.1.1
+- Alfresco Digital Workspace 1.2.0
+- Alfresco Process Workspace 1.3.4
+- Alfresco Sync Service 3.1.2
+
+#####Shared Services
+- Alfresco Identity Service 1.1
```
Comment (jamalkm) [inline README.md]: Alfresco Identity Management Service

**Comment by melahn** [comment]
The Digital Workspace version is 1.3.0

We should also include AGS 3.2.0

**Pull request merged**